cmd/kyma/kyma.go

```diff
@@ -68,6 +69,7 @@ Kyma CLI allows you to install, test, and manage Kyma.
 		console.NewCmd(console.NewOptions(o)),
 		upgrade.NewCmd(upgrade.NewOptions(o)),
 		create.NewCmd(o),
+		sync.NewCmd(o),
```

Comment:
The `sync` command was already added in line 88 together with the other function commands, no need to add it a second time.